src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  display: flex;
+  flex-shrink: 0;
+
+  &__list {
+    display: flex;
+    align-items: center;
+    padding: 0;
+
```

Comment:
do u really need this space? 
![image](https://user-images.githubusercontent.com/121224957/215507524-3e8e2fa2-88af-4d39-af00-0abad3f2496c.png)